src/components/layout/SwapLayout/Card/Card.tsx

```diff
@@ -216,9 +215,9 @@ export const CardTable = styled.table<{
       position: relative;
       display: grid;
       grid-template-columns: ${({ $columns }): string => $columns || `repeat(auto-fit, minmax(3rem, 1fr))`};
-      // grid-template-rows
+      grid-template-rows: dssl;
```

Comment:
???

Reply:
that was me testing linter, forgot to remove